### ISSUE TITLE
Change community site to Github pages / Jekyll

### ISSUE THREAD

**Issue by skotzko** [akkadotnet/getakka.net#11]
As I'm working on the docs and the community site, I think we can pretty easily simplify the setup to use what's already built into Github pages / Jekyll, without much effort. 

If we change, we get to keep all our content and layouts, but:
- we don't need a separate build server (drone.io)
- dont need to use assemble.io / grunt / nodeJS (they're wonderful tools, but IMHO they are overkill here)
- more devs can update docs / contribute to community site since many more people know Jekyll
- don't have to write custom Gruntfile additions to get basic things like "rebuild pages when I make changes to them in development"
- deploying community site changes is just a simple PR merge

@rogeralsing I'm proposing a switchover, and am happy to lead the effort. You okay with this? Is there a particular reason we need with the current setup? I would like to simplify this process and setup so more people can easily contribute / make updates without.

**Comment by rogeralsing** [comment]
No....

Jekyll is bugged beyond usage.
1. The Markdown renderer is freaking awful, there are 2 or 3 baked into GH pages, none renders our docs correctly. (read; they can break mid render and spit out a freaking mess and you will have to spend hours and hours to find what little combination of markdown syntax that caused it)
2. Syntax highlighting is broken, there are 2 or 3 of this also baked into GH pages, none renders partial code blocks correctly. 
   I spent weeks battling the Jekyll maintainers to fix these issues and the final answer was "we know syntaxhighlighting is a mess in jekyll"
3. Customization, Jekyll does not support plugins in GH pages, this was the main reason for switching _from_ jekyll, as the problems above could not be fixed in any way.
   We have special features to highlight error/info callouts in out markdown, we also apply auto links to words matching our pages.

There is no difference doing PR's with the current setup than with jekyll as a PR to this repo triggers a build transparently for the user, just as jekyll for GH pages does this when the repo changes.

In short, the quality of tooling on the JS side of things are far far greater than the Ruby counterparts.
And I'm pretty willing to bet some money on that there are more poeple that know NodeJS than Jekyll, so the arguments does not really hold imo.
